.github/workflows/clean-aws-resources.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: AMI Monthly Resources Cleaner
```

Comment:
This needs to be renamed.

Reply:
Nice catch. Resolving it since itts in the latest commit.